account_invoice_merge_purchase/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Merge invoices from purchase order
+==================================
+
+This module provides a compatibility between purchase order workflow and account invoice merge module
+
+
+Usage
+=====
+
+* Click on install button
```

Comment:
Remove this line (this should be in installation section - BTW, the module is auto-install)